cupy/cuda/cudnn.pyx

```diff
@@ -452,6 +452,23 @@ cpdef setTensor4dDescriptorEx(size_t tensorDesc, int dataType,
     check_status(status)
 
 
+cpdef getTensor4dDescriptor(size_t tensorDesc):
+    cdef DataType dataType
+    cdef int n
```

Comment:
How about `cdef int n, c, h, w ....` ?

Reply:
Thank you. Will update based on your comment.